StyleGuidelines.md

```diff
@@ -769,9 +769,13 @@ function Write-Nothing
 
 ## Parameters
 ### Correct Format for Parameter Block
-An empty parameter block should be displayed on its own line like this: ```param ()```
-Otherwise, the opening and closing parentheses should each be on their own line.
-All text inside the parameter block should be indented once.
+
+- An empty parameter block should be displayed on its own line like this: `param ()`.
+- An non-empty parameter block should have the opening and closing parentheses on their own line.
+- All text inside the parameter block should be indented once.
+- Every parameter must include regardless if it actually requires decoration: `[Parameter()]`.
```

Comment:
'Every parameter must include the '[Parameter()]' decoration regardless of whether it is actually used or not.'